apps/docs/content/components/RadioGroup/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Radio group
+description: Radio group is used to render a short list of mutually exclusive options.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+
+import anatomy from './images/anatomy.png'
+export default ComponentLayout
+
+## Usage
+
+Use radio group to allow users to select a single option from a short list of related options.
+
+### Orientation
+
+A vertical orientation makes options easier to visually scan. A horizontal orientation is not yet supported.
```

Comment:
Let's remove the last sentence as we provide an example in the next tab for it...

```suggestion
A vertical orientation makes options easier to visually scan.
```